src/tests/mod.rs

```diff
@@ -0,0 +1,2 @@
+#[cfg(test)]
+pub mod process;
```

Comment:
Could you explain why you have to put the #[cfg(test)] in 2 files please ?

Reply:
You right that's not needed here. I will remove it